changelog.dd

```diff
@@ -7,6 +7,8 @@ $(COMMENT Pending changelog for 2.069. This will get copied to dlang.org and
 $(BUGSTITLE Library Changes,
 
 $(LI $(RELATIVE_LINK2 json-encode-control-characters, `std.json` encodes string control characters.))
+$(LI The package $(STDMODREF experimental_ndslice, std.experimental.ndslice) was added.
+It is also availble at the $(LINK2 https://github.com/DlangScience/mir, Mir library).)
```

Comment:
typo: available
besides: maybe use "as part of" instead of "at"? sounds better to me